requirements.txt

```diff
@@ -4,4 +4,8 @@ flake8-quotes==3.3.1
 flake8-variables-names==0.0.5
 pep8-naming==0.13.2
 django-debug-toolbar==3.2.4
-django-crispy-forms==1.14.0
\ No newline at end of file
+django-crispy-forms==1.14.0
+django-bootstrap4==23.1
+crispy-bootstrap4==2022.1
```

Comment:
Are you sure you did the right task? It contains lots of code that is irrelevant for this task